app/src/main/res/layout-land/story_chapter_view.xml

```diff
@@ -32,7 +32,9 @@
       android:clickable="@{viewModel.chapterSummary.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES}"
       android:onClick="@{(v) -> viewModel.onExplorationClicked()}"
       app:cardCornerRadius="4dp"
-      app:cardElevation="@{viewModel.chapterSummary.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? @dimen/story_chapter_card_playable_state_elevation : @dimen/story_chapter_card_not_playable_state_elevation}">
+      app:cardElevation="@dimen/story_chapter_card_playable_state_elevation"
+      app:strokeColor="@color/colorPrimary"
```

Comment:
This is not fixed. If the card is locked then this border is grey. Check `Chapter-3` in this mock. https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/cce938d6-e079-4761-bac0-8bde22ac6348/

Reply:
I didn't notice that at all, sorry about that. Added grey border and updated screenshot above.